timemachine/potentials/jax_utils.py

```diff
@@ -99,7 +99,7 @@ def get_interacting_pair_indices_batch(confs, boxes, pairs, cutoff=1.2):
     assert max_n_neighbors > 0
 
     # sorting in order of [falses, ..., trues]
-    keep_inds = np.argsort(neighbor_masks, axis=1)[:, -max_n_neighbors:]
+    keep_inds = np.argsort(neighbor_masks, kind="stable", axis=1)[:, -max_n_neighbors:]
```

Comment:
(The uses of this one should be safe w.r.t. sort instability (since it's just used to partition "within cutoff" vs "outside cutoff"), but a good change regardless)

Reply:
I agree, just thought we should try and be consistent across the board.

Reply:
+1 for consistency